lab08-jacob/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Probably a few other basics that you want in here like `.DS_Store`, and the `data/` directory.